l10n_fr_ape/views/res_partner_industry_view.xml

```diff
@@ -0,0 +1,35 @@
+<odoo>
+
+<record id="res_partner_industry_view_form" model="ir.ui.view">
+    <field name="model">res.partner.industry</field>
+    <field name="inherit_id" ref="base.res_partner_industry_view_form"/>
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='name']" position="after">
+            <field name="code"/>
+        </xpath>
```

Comment:
```suggestion
        <field name="name" position="after">
            <field name="code"/>
        </field>
```

I find that not using xpath when it's not really required makes the code much more readable, don't you think?